ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```diff
@@ -290,15 +300,38 @@ run_command "oc version -o yaml"
 # private mirror registry host
 # <public_dns>:<port>
 MIRROR_REGISTRY_HOST=`head -n 1 "${SHARED_DIR}/mirror_registry_url"`
-echo "MIRROR_REGISTRY_HOST: ${MIRROR_REGISTRY_HOST}"
 # the proxy registry port 6001 for quay.io
 MIRROR_PROXY_REGISTRY_QUAY=`echo "${MIRROR_REGISTRY_HOST}" | sed 's/5000/6001/g' `
-echo "MIRROR_PROXY_REGISTRY_QUAY: ${MIRROR_PROXY_REGISTRY_QUAY}"
 # the proxy registry port 6002 for redhat.io
 MIRROR_PROXY_REGISTRY=`echo "${MIRROR_REGISTRY_HOST}" | sed 's/5000/6002/g' `
+
+# we don't set the proxy registy for the C2S and SC2S clusters, so use the default mirror registry port: 5000
```

Comment:
For L303-321:
https://github.com/openshift/release/blob/f82ff51b17ebcf407b909fb813f00a79782e24f3/ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh#L303-L321

I think we can make the logic more simple and easy to read, e.g. 

```bash
if [[ "${CLUSTER_TYPE:-}" =~ ^aws-s?c2s$ ]]; then
    # Use default registry
    echo "Using default registry, port 5000 for all"
    MIRROR_PROXY_REGISTRY_QUAY=${MIRROR_REGISTRY_HOST}
    MIRROR_PROXY_REGISTRY=${MIRROR_REGISTRY_HOST}
    mirror=1

else
    # Use proxy registry
    echo "Using proxy registry, port 6001/6002 for quay.io/redhat.io"
    # the proxy registry port 6001 for quay.io
    MIRROR_PROXY_REGISTRY_QUAY=`echo "${MIRROR_REGISTRY_HOST}" | sed 's/5000/6001/g' `
    # the proxy registry port 6002 for redhat.io
    MIRROR_PROXY_REGISTRY=`echo "${MIRROR_REGISTRY_HOST}" | sed 's/5000/6002/g' `
fi
```

Reply:
Thanks! But, what's the `CLUSTER_TYPE`? How can you get it?

Reply:
Which cluster resource returns the `aws-c2s` strings directly?

Reply:
`CLUSTER_TYPE` comes from `CLUSTER_PROFILE`

The cluster profiles for C2S/SC2S are `aws-c2s-qe`/`aws-c2s-qe`, the cluster types are `aws-c2s`/`aws-c2s`
You can always get `CLUSTER_TYPE` for C2S/SC2S clusters.